src/sentry/api/event_search.py

```diff
@@ -360,6 +374,31 @@ def visit_numeric_filter(self, node, children):
             )
             return self._handle_basic_filter(search_key, "=", search_value)
 
+    def visit_result_filter(self, node, children):
+        (search_key, _, operator, search_value) = children
+        operator = operator[0] if not isinstance(operator, Node) else "="
+        try:
+            search_value = SearchValue(int(search_value.text))
+        except ValueError:
+            raise InvalidSearchQuery("Invalid result query: %s" % (search_key,))
```

Comment:
```suggestion
            raise InvalidSearchQuery("Invalid aggregate query condition: %s" % (search_key,))
```
I don't think `result` is a term we use in the UI or docs to talk about aggregate functions.

Reply:
👍